### DISCUSSION TITLE
Estrutura para cursos online

### DISCUSSION THREAD

**Discussion by willianjusten** [frontendbr/forum#279] [Discussão]
Olá pessoal,

como a maioria deve saber, eu tenho criado [cursos](http://willianjusten.teachable.com/) na minha plataforma e tenho uma metodologia, mas sempre gosto de ouvir opiniões dos outros, seja para melhorar minhas coisas, seja para saciar minha curiosidade.

Hoje eu crio vídeos bem curtos (no máx 10 min) e separo os conteúdos por módulos, porque acho que isso facilita a pessoa a fixar os assuntos e não se cansar muito com vídeos grandes.

Mas queria saber como vocês preferem, como vocês conseguem absorver mais conteúdo em vídeo? Acham legal a forma de workshop ao vivo? Preferem vídeos gravados mesmo?  Ou vídeos maiores?

E quanto ao material? Preferem algo mais teórico com exemplos simples?
Ou algo mais prático e menos teórico?

Enfim, passem ae suas preferências =D

**Comment by brunopulis** [comment]
Fala @willianjusten, então eu prefiro cursos com vídeos curtos mesmo. Acredito que um workshop ao vivo dependendo do conteúdo é uma excelente escolha. 

Em relação ao material eu penso que deve ter um equilíbrio entre teoria/prática. Na teoria seria excelente colocar exemplos simples. Ter somente prática e pouca teoria do porquê eu não curto muito. Bom é isso! :)

**Comment by onhernandes** [comment]
Cara, em relação ao vídeos, eu prefiro como estão. Já tentei(tentei mesmo) assistir vídeo aulas enormes, de 15m, 30m e até 55m, sinceramente, não dá. Eu gosto muito das suas aulas como elas são agora haha

**Comment by eduardojmatos** [comment]
Se for pra aula via vídeo, eu curto mais os de baixa duração tbm!

Tem uma issue sobre método de estudo que o pessoal falou sobre estudo via vídeo: https://github.com/frontendbr/forum/issues/273

**Comment by vitor-mendrone-deel** [comment]
Sobre os vídeos eu fico com a maioria, prefiro os mais curtos, é difícil manter o foco por muito tempo em uma video aula. 

Aulas gravadas funcionam bem em cursos, workshops ao vivo funcionam mais pra apresentar algo bem específico. 

Aqui no Fastcommerce costumamos fazer palestras online para lojistas e muita gente abandona as palestras no meio. Os motivos são variados mas o principal é sempre o tempo. 

Já sobre material, acredito que seja legal ter bastante referências teóricas (tipo link pra artigos, dicas de livros, etc) mas durante o curso o ideal IMHO é que tudo seja mais prático. É o velho "Aprender fazendo" que faz bastante sentido pra mim.

**Comment by wellingtonpgp** [comment]
- Algo  algo mais pratico e menos teorico, com video aulas curtas.
- Ter tipo um lugar para digitar seus codigos e ver se esta correto.
- Exercicios pratico para aluno.
- Parte para sujestoes para o aluno dar o feedback do que esta achando do curso, no final de cada modulo.  :D

**Comment by eberhara** [comment]
@willianjusten se vc conseguir manter os vídeos curtos e focados em partes específicas assim como estão, está muito bom do ponto de vista prático.

**Comment by alisonmonteiro** [comment]
@willianjusten ainda não assisti nenhum curso teu, mas aulas menores, tipo as do Wes Bos, são muito boas. Misturando teoria e prática, sem enrolação. Aulas grandes demais muitas vezes perdem o foco. Abs!

**Comment by fdaciuk** [comment]
Boa @willianjusten! Eu curto vídeos mais curtos também ("curto vídeos curtos" :stuck_out_tongue_winking_eye:), mostrando na prática as partes teóricas.. tenho tentado fazer esse formato nos [meus cursos](http://blog.da2k.com.br/cursos/) também xD

**Comment by suissa** [comment]
Vídeos curtos

**Comment by raloliver** [comment]
Então, eu passo por essa dificuldade ao gravas vídeos aqui no trabalho, então gravamos, para o mesmo conteúdo, três vídeos: um de um minuto com a prática rápida e explicação direta, um mais detalhado onde eu apareço (o assunto é o mesmo, mas com um roteiro escrito antes) e um vídeo screen curto no formato que você já faz. Os hangouts é uma boa também, mas é legal ter uma pauta e não passar de uma hora pra não ficar chato e enfadonho.

**Comment by adrianoavelino** [comment]
Videos curtos são OS melhores. Práticas, exercí
cios apos OS videos com correcao tbm

**Comment by felipefialho** [comment]
Acho que sou a contramão do mundo.

Tenho uma certa resistência para assistir videos, isso vale para tudo, na profissão ou na vida pessoal. Sou muito hiperativo e não consigo continuar focado por muito tempo. Costumo ter mais facilidade para aprender com textos.

Então, baseando em pessoas como eu, acho que videos curtos são melhores :smile:

**Comment by jonasporto** [comment]
Eu gosto de vídeos curtos, porque acho tranquilo parar em um tópico, voltar dias depois e dar continuidade. 

Aproveitando... Uma feature que acho super importante, é a possibilidade de reproduzir os vídeos, alterando a velocidade para 1.5x ou 2x.
